### DISCUSSION TITLE
Syntax highlighting in source editor dialogs

### DISCUSSION THREAD

**Discussion by brettz9** [SVG-Edit/svgedit#796] [New feature request]
For SVG source editing (SVG XML) and for the foreignobject extension editing (XML), syntax highlighting as with ACE Editor or CodeMirror would be desirable.

**Comment by jggsus88** [comment]
I have experience integrating Ace editor in other applications. Can I have this issue?

**Comment by jfhenon** [comment]
A few questions: 
- Is Ace editor an heavy package? can you import it from npm? 
- Is there alternatives to Ace Editor (we just want to highlight the syntax)? 
- could we implement this as an extension so users would be able to choose the extension they prefer?
